packages/@glimmer/integration-tests/test/initial-render-test.ts

```diff
@@ -516,7 +545,7 @@ class Rehydration extends AbstractRehydrationTests {
       remoteParent: clientRemoteParent,
       remoteChild: clientRemoteChild,
     });
-    this.assertRehydrationStats({ nodesRemoved: 2 });
+    this.assertRehydrationStats({ nodesRemoved: 0 });
```

Comment:
This PR fixes rehydration for in-element, that is why this goes to expect no removed nodes.